charts/README.md

```diff
@@ -6,57 +6,92 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
 # Prerequisites
 * [Insall Helm version 3 or later](https://helm.sh/docs/intro/install/)
 
-# Install
-## Install CRDs
-clusterpedia requires CRD resources, which can be installed manually using kubectl, or using the `installCRDs` option when installing the Helm Chart
-> This way references the [cert-manager](https://cert-manager.io/docs/installation/helm/)
+Pull the Clusterpedia repository.
+
+> Currently, the chart has not been uploaded to the public charts repository.
 
-**Option 1: install CRDs with `kubectl`**
 ```bash
-$ kubectl apply -f ./_crds
+git clone https://github.com/clusterpedia-io/clusterpedia.git
+cd clusterpedia/charts
 ```
 
-**Option 2: install CRDs as part of the Helm release**
-To automatically install and manage the CRDs as part of your Helm release, you must add the `--set installCRDs=true` flag to your Helm installation command.
+Since Clusterpedia uses `bitnami/postgresql` and `bitnami/mysql` as subcharts of storage components, it is necessary to add the bitnami repository and update the dependencies of the clusterpedia chart.
+
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm dependency build
+```
+
+# Choose storage components
```

Comment:
These steps look like they are part of the installation steps, and the entire title format should look like：
```
## Prerequisites
## Install
### Choose a storage component
### Choose a installation or management mode for CRDs
### Check if you need to create a local PV
### Insatll Clusterpedia
### Final Check
### Create Cluster Auto Import Policy --- ClusterImportPolicy
## Uninstall
```

Reply:
we also need to add **Final Check** and **Create Cluster Auto Import Policy —— ClusterImportPolicy**.

ref https://clusterpedia.io/docs/installation/kubectl-apply/#final-check

`ClusterImportPolicy` also needs to be cleaned up:

If you have deployed ClusterImportPolicy then you need to clean up the ClusterImportPolicy resources first.
```bash
kubectl get clusterimportpolicy
```